### ISSUE TITLE
What dimensions should a full size image have?

### ISSUE THREAD

**Issue by iseulde** [WordPress/gutenberg#128]
At the moment the full size image takes up the whole window width, but we won't be able to do this in either wp-admin or calypso, unless we have some sort of full screen editor. Should we consider adding a frame (see single instance TinyMCE) to limit the size for a more realistic experience?

**Comment by jasmussen** [comment]
A good way to think of the editor _content_, is to think of it as a canvas. A white piece of paper that we have placed upon the wp-admin. This is what we've spent the most time on, so far.

Yes, it's smaller than the canvas your website will be shown on, but it is still a rectange. So there shouldn't be any reason we can't show full-bleed images full-bleed inside the editor?

Quick mockup:

![admin chrome sidebar open copy](https://cloud.githubusercontent.com/assets/1204802/23271287/405c7c08-f9f7-11e6-9a9e-d17e3aef2110.png)
